labelbox/schema/dataset.py

```diff
@@ -750,3 +755,35 @@ def _export(
         res = res[mutation_name]
         task_id = res["taskId"]
         return Task.get_task(self.client, task_id)
+
+    def upsert_data_rows(self, items: list[DataRowUpsertItem]) -> "Task":
+        chunk_size = 3
+        chunks = [
+            items[i:i + chunk_size] for i in range(0, len(items), chunk_size)
+        ]
+        manifest = {"source": "SDK", "item_count": 0, "chunk_uris": []}
+        for chunk in chunks:
+            manifest["chunk_uris"].append(self._create_descriptor_file(chunk))
```

Comment:
If we re-use this method we need to expose setting the file name; otherwise I believe it will just overwrite the file if there are multiple chunks.

Reply:
it won't overwrite it, an UUID is appended to the filename as a prefix